src/app/app.module.ts

```diff
@@ -34,9 +36,11 @@ import { QuestionPageComponent } from './components/practice/question-page/quest
     ClarityModule,
     BrowserAnimationsModule,
     ReactiveFormsModule,
-    HttpClientModule
+    HttpClientModule,
+    KatexModule
   ],
   providers: [],
   bootstrap: [AppComponent]
 })
 export class AppModule { }
+platformBrowserDynamic().bootstrapModule(AppModule);
```

Comment:
I don't think we need this as I was under the impression angular does this by default: [see their docs](https://angular.io/guide/bootstrapping#the-bootstrap-array)